web/thesauruses/python/exception_handling.json

```diff
@@ -0,0 +1,318 @@
+      "file_already_exists_exception_type",
+      "item_is_actually_directory_exception_type",
+      "item_is_not_a_directory_exception_type",
+      "cannot_access_file_exception_type",
+      "drive_disconnected_exception_type",
+      "file_no_longer_available_exception_type",
+      "file_timeout_exception"
+    ],
+    "Network Input/Output Errors/Exceptions": [
+      "base_network_exception_type",
+      "connection_refused_exception_type",
+      "connection_broken_exception_type",
+      "invalid_hostname_exception",
+      "cannot_establish_connection_exception_type",
+      "connection_timed_out_exception_type",
+      "bad_certificate_exception_type",
+      "expired_certificate_exception_type"
+    ],
+    "Operating System Errors/Exceptions": [
+      "child_process_could_not_be_created_exception_type",
+      "child_process_unexpected_ending_exception_type",
+      "process_not_found_exception_type"
+
+    ],
+    "Other Errors/Exception Types": [
+      "import_not_found_exception_type",
+      "keyboard_interrupt_exception_type",
+      "syntax_exception_type",
+      "uncaught_signal_exception_type"
+    ],
+    "Warnings": [
+      "base_warning_type",
+      "deprecation_warning_type"
+    ]
+  },
+  "exception_handling": {
+    "statement_to_import_exception_handling": {
+      "not-implemented": "true"
+    },
+    "are_exception_types_functions": {
+      "code": "No"
+    },
+    "are_exception_types_objects": {
+      "code": "Yes"
+    },
+    "are_exception_types_gotos": {
+      "code": "No"
+    },
+    "base_exception_type": {
+      "code": "BaseException"
+    },
+    "define_new_exception_type": {
+      "code": "class CustomError(Exception):\n    statements"
+    },
+    "try_or_operator": {
+      "code": "try:\n    statements\nexcept Exception:\n    statements"
+    },
+    "statement_to_start_trying_code": {
+      "code": "try:\n    statements"
+    },
+    "statement_to_start_catching_exceptions": {
+      "code": "except ExceptionType:\n    statements",
+      "comment": "instead of \"ExceptionType\" use respective Exception's class name you want to catch"
+    },
+    "statement_to_catch_generic_exception": {
+      "code": "except Exception:\n    statements"
+    },
+    "statement_to_catch_specific_exception": {
+      "code": "except ExceptionType:\n    statements",
+      "comment": "instead of \"ExceptionType\" use respective Exception's class name you want to catch"
+    },
+    "statement_to_catch_multiple_exceptions": {
+      "code": "except (ExceptionType1, ExceptionType2, ExceptionType3):\n    statements",
+      "comment": "instead of \"ExceptionType1\",\"ExceptionType2\",\"ExceptionType3\" use respective Exception's class name you want to catch. You can put as many Exception class names separated by commas as you want."
+    },
+    "statement_to_break_out_if_caught_exception": {
+      "not-implemented": "true"
+    },
+    "statement_to_stop_program_if_caught_exception": {
```

Comment:
You have "raise" below as a way to throw an error. This would be a way to actually stop/close the program if an exception was caught. I don't think Python has this either.

Reply:
I thought "raise" would throw an error and the program would stop if that error was not caught or we can close the program as we want by catching the error.
But, yes there is no specific built-in statement to stop the program if an exception is caught.